pyproject.toml

```diff
@@ -80,7 +80,7 @@ skip-magic-trailing-comma = false
 line-ending = "auto"
 
 [tool.mypy]
-exclude = ["examples", "venv"]
+exclude = ["examples", "venv", "cookbook"]
```

Comment:
remove examples here, I'm realizing I forgot to update this when I updated examples -> cookbook

Reply:
and cookbook